script/drupal-aws-cache.js

```diff
@@ -72,7 +71,7 @@ async function fetchCache() {
     options.buildtype === ENVIRONMENTS.LOCALHOST
       ? ENVIRONMENTS.VAGOVDEV
       : options.buildtype;
-  const cacheKey = getDrupalCacheKey(cacheEnv);
+  const cacheKey = await getDrupalCacheKey(cacheEnv);
```

Comment:
This is an async function that was previously resulting in unhandled promise rejections due to the `--fetch` command line arg not being recognized in `src/site/stages/build/options.js`.

It should `await`, but the problematic `--fetch` arg was also added (hackily) to the list of accepted options in that module.